packages/lib/src/components/Dragonpay/components/DragonpayVoucherResult/DragonpayVoucherResult.tsx

```diff
@@ -25,7 +25,7 @@ export default function DragonpayVoucherResult(props: DragonpayVoucherResultProp
             surcharge={surcharge && i18n.amount(surcharge.value, surcharge.currency)}
             voucherDetails={
                 [
-                    { label: i18n.get('voucher.expirationDate'), value: i18n.date(expiresAt) },
+                    { label: i18n.get('voucher.expirationDate'), value: i18n.dateTime(expiresAt) },
```

Comment:
Are we sure that the `expiresAt` will always be a date & time value? What happens if it's still just a date only?

Reply:
It will always be there since the backend is sending a ISO date. It could be wrong but then it's something on backend/acquiring side.